python/ray/data/read_api.py

```diff
@@ -1044,7 +1044,11 @@ def read_json(
         ...     "s3://anonymous@ray-example-data/log.json",
         ...     read_options=pajson.ReadOptions(block_size=block_size)
         ... )
-        Dataset(num_blocks=8, num_rows=1, schema={timestamp: timestamp[s], size: int64})
+        Dataset(
+           num_blocks=...,
```

Comment:
this does not work. sometimes it is in one line, and sometimes in multiple lines.

rather than printing the entire `Dataset` out, maybe just print out the `num_rows` and `schema`? and assert `num_blocks`?

or is there some way to force the environment to treat `num_blocks` to be a constant, and not dependent on environment?

Reply:
Yeah noticed that, will skip doc test in this case. In the future, we would always print out Dataset in multiple lines.